lib/qwerty/draw.rb

```diff
@@ -0,0 +1,11 @@
+module Qwerty
+  class Draw < Ruote::Participant
+    def on_workitem
+      workitem.fields['draw'] = {
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.